Example/SlideController/MainController.swift

```diff
@@ -44,10 +60,34 @@ class MainController {
             break
         }
     }
+    
+    lazy var removeAction: (() -> Void)? = { [weak self] in
+        guard let strongSelf = self else { return }
+        strongSelf.slideController.removeAtIndex(index: 0)
+    }
+    
+    lazy var insertAction: (() -> Void)? = { [weak self] in
+        guard let strongSelf = self else { return }
+        let page = SlidePageModel<PageLifeCycleObject>(object: PageLifeCycleObject())
+        let index = strongSelf.slideController.content.count == 0 ? 0 : strongSelf.slideController.content.count - 1
+        strongSelf.slideController.insert(object: page, index: index)
+        if index < strongSelf.slideController.content.count {
+            strongSelf.slideController.titleView.items[index].titleLabel.text = String(format: "page %d", strongSelf.pageIndex)
```

Comment:
Why we change label only for inserted page? Looks like labels for next pages must be updated too because they where shifted.